packages/browser/src/ClientAuthentication.ts

```diff
@@ -73,13 +73,14 @@ export default class ClientAuthentication {
   ): Promise<void> => {
     // In order to get a clean start, make sure that the session is logged out on login.
     await this.sessionInfoManager.clear(sessionId);
+
     return this.loginHandler.handle({
       sessionId,
       oidcIssuer: this.urlOptionToUrl(options.oidcIssuer),
       redirectUrl: this.urlOptionToUrl(options.redirectUrl),
       clientId: options.clientId,
       clientSecret: options.clientSecret,
-      clientName: options.clientId,
+      clientName: options.clientName || options.clientId,
```

Comment:
Just to be on the safe side - in case the client's name is `false` or something :)

```suggestion
      clientName: options.clientName ?? options.clientId,
```